### PR TITLE
feat:  add information to the `/apple` page for the macOS standalone client user

### PR DESCRIPTION
**Pull request by CNLHC**
- [x] read the [CONTRIBUTING guidelines](README.md#contributing)
- [x] raised a GitHub issue or discussed it on the projects chat beforehand
- [ ] added unit tests
- [ ] added integration tests
- [ ] updated documentation if needed
- [ ] updated CHANGELOG.md

#912 

Besides the new configuration, This PR  introduces the `embed` package to this project.

The verbose HTML files are hard to edit in the go source code. Moving them to separate files has many benefits including a better editing experience and the potential to add more advanced features to the front (e.g. prettier UI or interactive login)

### CODE REVIEW

File: platform_config.go [hunk 1]
```diff
@@ -6,56 +6,25 @@ import (
...
 	textTemplate "text/template"
 
+	_ "embed"
+
 	"github.com/gofrs/uuid"
 	"github.com/gorilla/mux"
 	"github.com/rs/zerolog/log"
 )
 
+//go:embed static/apple.html
```
Comment (kradalby) [inline platform_config.go:16]: can you call this folder `templates/` instead of static?

**Review by kradalby — COMMENTED** [review]
One small nit, and some linting errors, otherwise good.

File: platform_config.go (hunk 1 above)
Comment (CNLHC) [reply platform_config.go:16]: sure

**Comment by kradalby** [comment]
Please also have a look at the Prettier linter error

**Comment by CNLHC** [comment]
> Please also have a look at the Prettier linter error

done

meanwhile, I just found currently seems not formatted via `make fmt`, simply using this to format code will cause a lot of changes in go source code.

**Review by kradalby — APPROVED** [review]

**Pull request merged**